api/lib/domain/usecases/authenticate-user.js

```diff
@@ -20,18 +21,38 @@ function _checkUserAccessScope(scope, user) {
   }
 }
 
+async function _checkScoUserAccessCertifScope({ scope, user, certificationCenterMembershipRepository }) {
+  const userBelongsToSco = await _doesUserBelongsToScoCenter({ user, certificationCenterMembershipRepository });
+  const blockingAccessToScoUserFeatureToggleIsOn = config.featureToggles.certifBlockingScoUserAccess;
+  const scopeIsPixCertif = scope === apps.PIX_CERTIF.SCOPE;
+
+  if (scopeIsPixCertif &&
+        blockingAccessToScoUserFeatureToggleIsOn &&
+        userBelongsToSco)
+  {
+    throw new ForbiddenAccess(apps.PIX_CERTIF.USER_SCO_BLOCKED_CERTIFICATION_MSG);
+  }
+}
+
+async function _doesUserBelongsToScoCenter({ user, certificationCenterMembershipRepository }) {
+  const certificationCenterMemberships = await certificationCenterMembershipRepository.findByUserId(user.id);
+  return certificationCenterMemberships && certificationCenterMemberships.some((membership) => membership.certificationCenter.isSco);
+}
+
 module.exports = async function authenticateUser({
   password,
   scope,
   source,
   tokenService,
   username,
   userRepository,
+  certificationCenterMembershipRepository,
 }) {
   try {
     const foundUser = await authenticationService.getUserByUsernameAndPassword({ username, password, userRepository });
 
     if (!foundUser.shouldChangePassword) {
+      await _checkScoUserAccessCertifScope({ scope, user: foundUser, certificationCenterMembershipRepository });
       _checkUserAccessScope(scope, foundUser);
```

Comment:
Inverser l'ordre des appels : 
```suggestion
      _checkUserAccessScope(scope, foundUser);
      await _checkScoUserAccessCertifScope({ scope, user: foundUser, certificationCenterMembershipRepository });
```